app/models/manageiq/providers/openstack/network_manager/cloud_network.rb

```diff
@@ -20,6 +24,229 @@ class ManageIQ::Providers::Openstack::NetworkManager::CloudNetwork < ::CloudNetw
     end
   end
 
+  def self.params_for_create(ems, ems_id, cloud_network_id)
+    unless defined?(cloud_network_id).nil?
+      cloud_network_id = false
+    end
```

Comment:
I think you just need to check `:isDisabled => cloud_network_id.present?` instead of all this